src/sections/EpisodeListSection.svelte

```diff
@@ -17,6 +17,9 @@
         {/if}
       </li>
     {/each}
+    <li>
+      <a href="?randomEpisode">Random episode</a>
+    </li>
```

Comment:
I think this should instead be a `<button>` that calculates an episode title and then navigates to that, instead of an anchor tag that links to a specific page that displays random content.

This because then the randomized episodes gets stored in your browser's history, whereas `?randomEpisode` would not add history items when switching between random songs.

To not have it reload the page, look at the implementation in `Link.svelte` where it sends a `onpushstate` window event, that then `App.svelte` listens to, to change the loaded episode or page without a page reload.